lang/cs/Org.Apache.REEF.Tests/Functional/IMRU/TestTaskExceptions.cs

```diff
@@ -185,7 +185,7 @@ public void OnNext(IFailedTask value)
                     }
 
                     Assert.Equal(InnerExceptionMessage, value.AsError().InnerException.Message);
-                    Assert.True(value.AsError().InnerException is System.SystemException);
+                    Assert.True(value.AsError().InnerException is Exception);
```

Comment:
We need a particular exception type for the test otherwise, any exception would meet the assert.True.